packages/cms/store/resource.ts

```diff
@@ -110,21 +110,32 @@ export const useResourceStore = create<ResourceState & ResourceMethods>(
       set({
         filters: get().filters.filter(
           activeFilter =>
-            activeFilter.field.databaseField === filter.field.databaseField
+            activeFilter.field.databaseField !== filter.field.databaseField
         )
       })
     },
 
     async fetchTableData(params: TableDataParams) {
-      const { resource } = get()
+      const { resource, filters } = get()
       const { page, perPage, sort, sortField, search } = params
 
       return window.Tensei.api.get(`/${resource?.slug}`, {
         params: {
           page,
           perPage,
           ...(sortField && { sort }),
-          ...(search && { search })
+          ...(search && { search }),
+          ...(filters.length && {
+            where: {
```

Comment:
This `where` should be optional, only added when there is at least one filter.

Reply:
it is optional?  it doesn't send a request to the DB if there is no filter applied.
`filters. length &&` checks the array.